services/QuillLMS/app/lib/utils/numeric.rb

```diff
@@ -26,7 +26,7 @@ def self.human_readable_time_to_seconds(time)
     (time.split(":")[0].to_i * 60) + (time.split(":")[1].to_i)
   end
 
-  def self.safe_division(numerator, denominator, fallback = 0)
+  def self.safe_division(numerator, denominator, fallback = nil)
```

Comment:
This actually seems like a better default than 0.  Not sure why I didn't think of nil.